src/functions/memo-management/drive-memos/get-content-pdf/index.ts

```diff
@@ -1,16 +1,21 @@
 import { handlerPath } from "@libs/handler-resolver";
 
-export default {
-  handler: `${handlerPath(__dirname)}/handler.main`,
-  events: [
-    {
-      httpApi: {
-        method: "get",
-        path: "/google-drive/memo-content",
-        authorizer: {
-          name: "homeKeycloakAuthorizer"
+// NOTE: This lambda is marked as inactive in the OpenAPI spec (x-status: inactive).
+const isInactive = process.env.SKIP_INACTIVE_LAMBDAS === "true";
```

Comment:
No sure we should use an env for this, rather can't we just set isInactive to true directly here?